consensus/consensus-types/src/wrapped_ledger_info.rs

```diff
@@ -0,0 +1,95 @@
+// Copyright © Aptos Foundation
+// Parts of the project are originally copyright © Meta Platforms, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::vote_data::VoteData;
+use anyhow::{ensure, Context};
+use aptos_types::{
+    aggregate_signature::AggregateSignature,
+    block_info::BlockInfo,
+    ledger_info::{LedgerInfo, LedgerInfoWithSignatures},
+    validator_verifier::ValidatorVerifier,
+};
+use serde::{Deserialize, Serialize};
+use std::fmt::{Display, Formatter};
+
+/// This struct is similar to QuorumCert, except that the verify function doesn't verify vote_data.
+/// vote_data and consensus_data_hash inside signed_ledger_info could be replaced with dummy values.
+#[derive(Deserialize, Serialize, Clone, Debug, Eq, PartialEq)]
+pub struct WrappedLedgerInfo {
```

Comment:
So it is still compatible, right?

Reply:
Yes. This PR is compatible with the `main` branch. `WrappedLedgerInfo` and `QuorumCert` has the same structure.